src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    operate(action, value) {
+      action(value);
+
+      return this;
+    },
+    add(value) {
+      result += value;
+
+      return this;
+    },
+    subtract(value) {
+      result -= value;
+
+      return this;
+    },
+    multiply(value) {
+      result *= value;
+
+      return this;
+    },
+    divide(value) {
+      result /= value;
+
+      return this;
+    },
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics